src/Core/Feature.js

```diff
@@ -370,7 +376,11 @@ export class FeatureCollection extends THREE.Object3D {
         this.mergeFeatures = options.mergeFeatures === undefined ? true : options.mergeFeatures;
         this.size = options.structure == '3d' ? 3 : 2;
         this.filterExtent = options.filterExtent;
-        this.style = options.style;
+        this.baseAltitude = {
+            fill: (options.style?.fill?.base_altitude) || baseAltitudeDefault,
+            stroke: (options.style?.stroke?.base_altitude) || baseAltitudeDefault,
+            point: (options.style?.point?.base_altitude) || baseAltitudeDefault,
+        };
```

Comment:
This move is debatable, it's out PR scope, could you move in other PR?

Reply:
It's not out of scope as it is need if we want to be able to use functions to set the feature.style which is exactly what is been done in this PR.